src/Compilers/CSharp/Portable/CSharpParseOptions.cs

```diff
@@ -86,6 +86,7 @@ public override IEnumerable<string> PreprocessorSymbolNames
             kind: other.Kind,
             preprocessorSymbols: other.PreprocessorSymbols)
```

Comment:
I'm not certain i even see the point of this constructor.  Parse options are immutable.  Why do we need a parse options constructor that just copies the values out of another parse options?